File: 프로그래머스/0/181933. flag에 따라 다른 값 반환하기/flag에 따라 다른 값 반환하기.js
function solution(a, b, flag) {
    let answer = 0;
    
    if (flag === true) {
        answer = a + b;
    } // 만약 flag가 true라면 a+b를 반환하고 
    else {
        answer = a - b;
    } // 만약 flag가 false라면 a-b를 반환한다
    return answer;
}

File: 프로그래머스/unrated/181933. flag에 따라 다른 값 반환하기/flag에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int a, int b, boolean flag) {
        int answer = 0;
        
        if (flag == true) {
            answer = a + b;
        }   // 만약 flag가 true라면 a+b를 반환하고
        else {
            answer = a - b;
        }   // 만약 flag가 false라면 a-b를 반환한다
        return answer;
    }
}